javascript/core/recipes-i/round-numbers-to-n-decimals.md

```diff
@@ -1,31 +1,18 @@
 ---
 author: catalin
 
-levels:
-
-  - basic
-
-  - beginner
-
 type: normal
 
 category: how to
 
-standards:
-
-  javascript.write-expressions.0: 10
-
-  javascript.evaluate-expressions.3: 10
-
 aspects:
   - introduction
   - workout
 
 
-
 ---
 
-# Round numbers to `N` decimals 
+# Round Numbers to `N` Decimals
```

Comment:
is this a new headline convention? we capitalize every non-filler word?

Reply:
I've been trying to put all headlines in title case, using the apa guidelines. I think this looks better, but also it is just a preference